dev-env/aet-swarm-dev.yml

```diff
@@ -95,9 +95,9 @@ services:
       - hub
       - activemq
     volumes:
-      - ./bundles:/aet/core/bundles
```

Comment:
This should remain as `core`. Custom bundles are additional bundles. Core bundles are those which are built as a release artifact form this repository.
If change like this remains, you would have doubled all core bundles.